### ISSUE TITLE
Illegal or missing directory 'share'

### ISSUE THREAD

**Issue by eserte** [kjetilk/p5-rdf-ns-curated#2]
Build fails (freebsd, linux; various perl versions):

```
Output from '/usr/perl5.22.0p/bin/perl Makefile.PL':

Illegal or missing directory 'share' at /usr/perl5.22.0p/lib/site_perl/5.22.0/File/ShareDir/Install.pm line 105.
        File::ShareDir::Install::_add_dir(HASH(0x4302dea0), "share") called at /usr/perl5.22.0p/lib/site_perl/5.22.0/File/ShareDir/Install.pm line 42
        File::ShareDir::Install::install_share() called at Makefile.PL line 137
```

**Comment by kjetilk** [comment]
huh...? So, the directive in dist.ini might have worked anyway...
I removed that from the code, I'll roll a new release.

**Comment by kjetilk** [comment]
Should be fixed by the upload of 0.002 just now.
